packages/components/src/components/dropdown/_dropdown.scss

```diff
@@ -66,6 +66,24 @@
     }
   }
 
+  .#{$prefix}--dropdown--xl {
+    height: rem(48px);
+    max-height: rem(48px);
+  }
+
+  .#{$prefix}--dropdown--xl .#{$prefix}--dropdown__arrow {
+    top: rem(16px);
+  }
+
+  .#{$prefix}--dropdown--sm {
+    height: rem(32px);
```

Comment:
Is it safe to set `height` and `max-height` if default text size changes? Should we use `min-height`?